core/rawdb/accessors_chain.go

```diff
@@ -672,7 +672,7 @@ func DeleteReceipts(db ethdb.KeyValueWriter, hash common.Hash, number uint64) {
 type storedReceiptRLP struct {
 	PostStateOrStatus []byte
 	CumulativeGasUsed uint64
-	Logs              []*types.LogForStorage
+	Logs              []*types.Log
```

Comment:
just a nitpick and you can ignore it

Is it possible to re-use the definition in core/types package soomehow?

Reply:
Maybe we can mark a TODO here.

Reply:
Do you have an idea how to do it without exporting an additional type in core/types?

Reply:
It would be kind of nice to remove the `DecodeRLP` method of `receiptLogs`. I think you could just rename the type `storedReceiptRLP` to `receiptLogs`. It would save a lot of reflect indirection during decoding.